docs/user/geoserver.rst

```diff
@@ -0,0 +1,92 @@
+GeoMesa GeoServer Plugins
+=========================
+
+Background
+----------
+
+When you install the GeoMesa plugins for GeoServer, you get access to the
```

Comment:
gone entirely - it hasn't been supported for a while, it would need some work to get it working again.